tests/Unit/Time/Test_History.cpp

```diff
@@ -795,6 +796,29 @@ void test_history_assertions() {
 
 #endif  // SPECTRE_DEBUG
 }
+
+void test_history_output() {
+  TimeSteppers::History<double> history(2);
+  const Slab slab(0.0, 1.0);
+  history.insert(TimeStepId(true, 0, slab.start()), 0.0, 1.0);
+  const auto step_time = history.back().time_step_id.step_time();
+  const auto step_size = slab.duration() / 4;
+  history.insert(TimeStepId(true, 0, step_time, 1, step_size,
+                            (step_time + slab.duration() / 4).value()),
+                 4.0, 40.0);
+  std::string expected_output = MakeString{}
```

Comment:
[optional] This could just be a string literal; no need for `MakeString`.  Also could be const.

Reply:
@wthrowe @nilsvu squashed in this change and rebased